/*
 * File:   MainWindow.cpp
 * Author: myww749
 *
 * Created on 5 September 2015, 9:31 PM
 */

#include "MainWindow.h"

MainWindow::MainWindow() {
    widget.setupUi(this);
}

void MainWindow::selectFile() {
    
}

void MainWindow::retrieveVersionData() {
    
}

void MainWindow::createFirstVersion() {
    
}

void MainWindow::saveCurrent() {
    
}

void MainWindow::selectEntry() {
    
}

bool MainWindow::showComment() {
    
}

bool MainWindow::retrieveVersion() {
    
}

void MainWindow::setReferenceVersion() {
    
}
        
MainWindow::~MainWindow() {
}
